doc-Managing_Infrastructure_and_Inventory/topics/Tagging_Orchestration_Stacks.adoc

```diff
@@ -0,0 +1,11 @@
+= Tagging Orchestration Stacks
+
+Apply tags to orchestration stacks to categorize them together at the same time. You can create tags using the instructions in the https://access.redhat.com/documentation/en/red-hat-cloudforms/4.1/general-configuration/general-configuration[General Configuration] guide.
```

Comment:
@dayleparker @suyogsainkar 

Hmm, do we need this reference? It doesn't really point to a specific section, and I think we should consider even adding the procedure for creating tags to the same guide or collecting all relevant sections in the same place at some stage, for example.

What do you think?